.github/workflows/release.yml

```diff
@@ -48,8 +59,16 @@ jobs:
           tag="${GITHUB_REF##*/}"
 
           hub release create
-          -a iso/alpine-lima-clm-${ALPINE_VERSION}-aarch64.iso
-          -a iso/alpine-lima-clm-${ALPINE_VERSION}-aarch64.iso.sha512sum
-          -a iso/alpine-lima-clm-${ALPINE_VERSION}-x86_64.iso
-          -a iso/alpine-lima-clm-${ALPINE_VERSION}-x86_64.iso.sha512sum
+          -a iso/alpine-lima-clmp-${ALPINE_VERSION}-aarch64.iso
```

Comment:
since this is colima specific, we can modify the name to something clearer.
```
alpine-colima-podman-${ALPINE_VERSION}-aarch64.iso
alpine-colima-containerd-${ALPINE_VERSION}-aarch64.iso
alpine-colima-docker-${ALPINE_VERSION}-aarch64.iso
```
And so on...

Reply:
I think there was something that the image name can only be a maximum of X characters, that's why the edition should be less than 5 characters. But I need to double check.